README.md

```diff
@@ -35,19 +35,40 @@ See [Pyspark](examples/Pyspark Kernel.ipynb) and [Spark](examples/Spark Kernel.i
 
 1. Install the library
 
-        pip install sparkmagic
+        `pip install sparkmagic`
```

Comment:
You don't need the backticks for teletype, just putting 4 spaces in front of the code is fine.  (See the current [README](https://github.com/jupyter-incubator/sparkmagic/blob/master/README.md))